pp_netlib/functions.py

```diff
@@ -1,6 +1,11 @@
 ########################
 ####   .CONSTRUCT   ####
 ########################
+from ast import Index
```

Comment:
This looks like it has been deprecated since python3.9. What is it used for? Can you find an alternative?

Reply:
it was an automatic import added by the code editor, not used